### DISCUSSION TITLE
Check if Windows PC is sleeping

### DISCUSSION THREAD

**Discussion by Macrox14** [AlexGustafsson/homebridge-wol#234] [General]
Hello, the plugin works very well.
Just want to know if is possible to verify if the Windows PC is sleeping.
This is because I've made a Shortcut which only turn off my outlet if my PC is OFF, but I don't want to shutdown the outlet if the PC is sleeping.

**Comment by AlexGustafsson** [comment]
You can probably solve this by using a ping command which uses SSH to the windows PC and then performs some check. Note that this entirely depends on how the PC behaves when it's sleeping. It's not impossible that it simply will not respond to anything. Then, it's impossible to know if it's sleeping or entirely off. You'll probably have better luck finding more information about this on forums like the superuser stack exchange. Basically, anything you can run manually as a command in your shell can be made to work with this plugin. Closing.
